src/index.html

```diff
@@ -3,11 +3,271 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/favicon.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
+
   <body>
-    <h1>Miami</h1>
+
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <div class="logo__container">
+          <a class="logo navigation__item" href="#">
+            <img
+              class="logo__image"
+              src="/images/logo-bose.svg"
+              alt="logo bose">
+          </a>
+        </div>
+
+        <div class="icons__container">
+          <a class="phone navigation__item" href="tel:+1 234 555-55-55">
+            <span class="phone__label">+1 234 555-55-55</span>
+            <img
+              class="phone__icon"
+              src="/images/phone.svg"
+              alt="phone">
+          </a>
+          <a class="burger navigation__item" href="#menu">
+            <span class="burger__label">Open menu</span>
+          </a>
+        </div>
+      </nav>
+
+      <h1 class="title title--decorated header__title">
+        Your music shines.
+        <br>
+        The world shades.
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <nav class="navigation menu__navigation">
+        <a class="logo navigation__item" href="#">
+          <img
+            class="logo__image"
+            src="/images/logo-bose.svg"
+            alt="logo bose">
+        </a>
+        <a class="close navigation__item" href="#home">
+          <img
+            class="close__icon"
+            src="/images/close.svg"
+            alt="close">
+        </a>
+      </nav>
+
+      <nav class="links menu__links">
+        <ul class="links__list">
+          <li class="links__item"><a href="#home" class="links__link">Home</a></li>
```

Comment:
Think you should also fix formatting here, to be like:

```suggestion
          <li class="links__item">
            <a href="#home" class="links__link">Home</a>
          </li>
```

Then will be more readable :)